common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -389,16 +389,8 @@ div.problem {
       }
     }
 
-    .unanswered {
-      p.status.drag-and-drop--status {
```

Comment:
This CSS is not working (with the old drag and drop problem) after the change to status-indicator class. And I don't see any need to have it. When the problem is "unanswered", there is nothing shown in this region.

Finally, this problem type is deprecated, so risk of breaking something is very low.

https://jsinput.sandbox.edx.org/courses/course-v1:jsinput+jsinput+jsinput/courseware/36ab1a99540048dfbb5b9af9218f5569/5d6e37b8d2da4609920859da9bfa57dc/2?activate_block_id=block-v1%3Ajsinput%2Bjsinput%2Bjsinput%2Btype%40vertical%2Bblock%40cd92c5354d1a46b39de86d5a63b793f6